go/common/crypto/signature/signers/ledger/ledger_signer.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
 	ledgerCommon "github.com/oasisprotocol/oasis-core/go/common/ledger"
+	ledger "github.com/zondax/ledger-oasis-go"
```

Comment:
This should be grouped in the above group of third-party imports. Also there already seems to be a `ledger` import above that points to `github.com/zondax/ledger-go` so you probably need to rename this one.

Reply:
Yeah, mi bad, I'll delete the `ledger-oasis-go` since is not necessary after doing the requested fix below.